src/components/cards/cards.js

```diff
@@ -18,46 +31,54 @@ class Cards extends Component {
         this.convertManaToSymbol.bind(this);
         this.getAddRemoveButtons.bind(this);
         this.addRemoveCard.bind(this);
-        this.toggleCardList.bind(this);
     }
 
     getCardsFromSet = () => {
         const sortedCards = this.props.cards.sortedCards;
 
         if (sortedCards.land) {
-            return Object.entries(sortedCards).map(([identity, cardSubset]) => {
-                if (identity === 'land') {
-                    return this.processLands(cardSubset);
+            return(<Accordion allowMultiple>
```

Comment:
This was a selling point, there's at least a bunch of decent functionality for free here